### PR TITLE
Fix/bug fixing

### PR DESCRIPTION
**Pull request by juliswer**

### CODE REVIEW

**Review by GimerStackIt — DISMISSED** [review]

File: src/components/item-details/bid-tab/bid-tab.tsx [hunk 1]
```diff
@@ -84,35 +85,47 @@ const ownerHard: any = [
 
 export const BidTab = ({ className, item }: Props) => {
   const canIList = useAppSelector((state) => selectCanIList(state, item));
+  const [selectedTab, setSelectedTab] = useState('nav-details' as string);
```
Comment (mmiranda-stackit) [inline src/components/item-details/bid-tab/bid-tab.tsx:88]: I guess typescript can infer the type (no need to say 'as string')

File: src/pages/item/[itemId].tsx [hunk 2]
```diff
@@ -36,25 +36,28 @@ const ProductDetails = ({ itemId, metaData }: Props) => {
   const formattedAddress = (address: string) =>
     `${address.substring(0, 7)}...${address.substring(address.length - 4, address.length)} - NFT`;
 
+  const pageTitleWindow = item
+    ? `${item?.name} #${item?.itemId.slice(0, 5)} - Jhon Doe | LEDA`
```
Comment (mmiranda-stackit) [inline src/pages/item/[itemId].tsx:40]: Let's just remove the fake data. We are going to prod soon and we will do this clean up anyways

File: src/containers/item-details/item-details.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
 
   const isOwner: boolean = address === item.owner.address;
 
-  const priceLabel = isOwner ? 'You own this NFT' : 'Buy it now for';
+  const priceLabel: string = isOwner ? 'You own this NFT' : 'Buy it now for';
```
Comment (mmiranda-stackit) [inline src/containers/item-details/item-details.tsx:26]: ts will infer the type I guess.

File: src/components/item-details/bid-tab/listing-tab-content.tsx [hunk 4]
```diff
@@ -28,14 +29,19 @@ export const ListingTabContent = ({ item }: Props) => {
...
-    dispatch(
-      listItem({
-        price,
-        tokenId: item.tokenId,
-        itemId: item.itemId,
-        ownerAddress: item.owner.address,
-      })
-    );
+    try {
+      await dispatch(
```
Comment (mmiranda-stackit) [inline src/components/item-details/bid-tab/listing-tab-content.tsx]: Please don't add logic related to actions in components. Leverage this logic to thunks and reducers

File: src/components/item-details/bid-tab/listing-tab-content.tsx [hunk 5]
```diff
@@ -28,14 +29,19 @@ export const ListingTabContent = ({ item }: Props) => {
...
+        listItem({
+          price,
+          tokenId: item.tokenId,
+          itemId: item.itemId,
+          ownerAddress: item.owner.address,
+        })
+      );
+      setSelectedTab('nav-details');
+    } catch (error) {
+      console.log(error);
```
Comment (mmiranda-stackit) [inline src/components/item-details/bid-tab/listing-tab-content.tsx]: This is not helping at all

File: src/containers/item-details/item-details.tsx [hunk 6]
```diff
@@ -48,7 +49,7 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
                 likeCount={item.likes}
                 itemId={item.itemId.slice(0, 4)}
               />
-              {item.price && (
+              {item.price && item.status !== 0 ? (
```
Comment (mmiranda-stackit) [inline src/containers/item-details/item-details.tsx:59]: Please do not use hardcoded values. There is a ItemStatus enum available. If it happens to not exist, please create it and use it instead

File: src/containers/item-details/item-details.tsx [hunk 7]
```diff
@@ -21,6 +21,8 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
 
   const isOwner: boolean = address === item.owner.address;
 
+  const isAuthor: boolean = address === item.author.address;
```
Comment (mmiranda-stackit) [inline src/containers/item-details/item-details.tsx:24]: It is cool to explicit type the data types, but on this case this is obvious and not necessary. It is like saying `const a:number = 2;`

File: src/containers/item-details/item-details.tsx [hunk 8]
```diff
@@ -49,6 +51,11 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
                 likeCount={item.likes}
                 itemId={item.itemId.slice(0, 4)}
               />
+              {isAuthor && (
+                <h6 className="bid d-flex flex-row align-items-center gap-2 my-4">
+                  You&apos;ve created a incredible NFT
```
Comment (mmiranda-stackit) [inline src/containers/item-details/item-details.tsx:56]: Change ` You&apos;ve created a incredible NFT` by  `You&apos;ve created an incredible NFT`

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]
Left some comments

**Review by mmiranda-stackit — APPROVED** [review]
looks good

**Pull request merged**